app/src/proprietary/java/org/jellyfin/mobile/player/cast/Chromecast.java

```diff
@@ -33,7 +33,7 @@ public final class Chromecast implements IChromecast {
     /**
      * Object to control the media.
      */
-    private ChromecastSession media;
+    private ChromecastSession chromecastSession;
```

Comment:
changed the variable name to be more "descriptive" as to what this variable actually was. For the longest time I got a little confused and thought this was the actual media, not the session.